### PR TITLE
Cors will

### PR DESCRIPTION
**Pull request by wjm5004**
this should ignore cors errors on current endpoints

### CODE REVIEW

File: api/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Flask==1.1.1
 Flask-RESTful==0.3.7
 gunicorn==19.9.0
-flask_cors
+Flask-Cors==3.0.8
```
Comment (wjm5004) [inline api/requirements.txt]: nice job man, i really like that addition, youre a genius

File: .gitignore [hunk 2]
```diff
@@ -7,3 +7,5 @@ jsconfig.json
 *.pyc
 api/rest/__pycache__/C
 __pycache__/
+.idea/
+.swp
```
Comment (wjm5004) [inline .gitignore]: good pick up, you deserve a raise

File: api/rest/team_routes.py [hunk 3]
```diff
@@ -2,11 +2,13 @@
 from api import db
 from rest.models import Team
 from rest.models import users_schema
+from flask_cors import cross_origin
```
Comment (wjm5004) [inline api/rest/team_routes.py]: gucci gang af

File: api/rest/team_routes.py [hunk 4]
```diff
@@ -2,11 +2,13 @@
...
 
 team = Blueprint('team', __name__)
 
 
 @team.route('/team/add', methods=['POST'])
+@cross_origin()
```
Comment (wjm5004) [inline api/rest/team_routes.py]: coder is code

File: api/rest/user_routes.py [hunk 5]
```diff
@@ -28,13 +30,15 @@ def add_user():
 
 
 @user.route('/user/all', methods=['GET'])
+@cross_origin()
```
Comment (wjm5004) [inline api/rest/user_routes.py]: *whistle*

File: api/requirements.txt (hunk 1 above)
Comment (sp1kedshell) [reply api/requirements.txt]: code code

File: .gitignore (hunk 2 above)
Comment (sp1kedshell) [reply .gitignore]: code code

File: api/rest/team_routes.py (hunk 3 above)
Comment (sp1kedshell) [reply api/rest/team_routes.py]: looks bueno

File: api/rest/team_routes.py (hunk 4 above)
Comment (sp1kedshell) [reply api/rest/team_routes.py]: code looks code

File: api/rest/user_routes.py (hunk 5 above)
Comment (sp1kedshell) [reply api/rest/user_routes.py]: yeet

**Pull request merged**